webclient/app/src/app/commons/entityDialog/EntityDialog.jsx

```diff
@@ -139,8 +145,12 @@ function EntityDialog(props) {
         setIsSaving(true);
         handleSave(entity)
             .then(() => {
-                onClose()
-                setIsSaving(false);
+                entityRest.findAllEntitiesByApp(appId)
+                    .then((response) => {
+                        handleUpdateEntities(response.data);
+                        onClose();
+                        setIsSaving(false);
+                    })
             })
```

Comment:
```suggestion
                return entityRest.findAllEntitiesByApp(appId)
            })
            .then((response) => {
                        handleUpdateEntities(response.data);
                        onClose();
                        setIsSaving(false);
                    })
```

Reply:
Why I should return a promise to an onClick call?